cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiTest.java

```diff
@@ -279,4 +282,34 @@ public void testChangeLoggers(VertxTestContext context) throws InterruptedExcept
                             async.flag();
                         }))));
     }
+
+    @IsolatedTest
+    public void testChangeParentLoggerLevel(VertxTestContext context) {
+        String desiredLogging = "log4j.rootLogger=TRACE, CONSOLE\n" +
+                "log4j.logger.io.org=WARN\n" +
+                "log4j.logger.io.org.com=FATAL";
+
+        desiredLogging = Util.expandVars(desiredLogging);
+        OrderedProperties ops = new OrderedProperties();
+        ops.addStringPairs(desiredLogging);
+
+        KafkaConnectApiImpl client = new KafkaConnectApiImpl(vertx);
+
+        Map<String, Map<String, String>> fetched = new HashMap();
+        fetched.put("root", singletonMap("level", "INFO"));
+        fetched.put("io.org", singletonMap("level", "WARN"));
+        fetched.put("io.org.com", singletonMap("level", "FATAL"));
+
+        // root logger level changed
+        assertTrue(client.parentLogLevelChanged(fetched, ops, Map.entry("io.org.com", "WARN")));
+
+        fetched = new HashMap();
+        fetched.put("root", singletonMap("level", "TRACE"));
+        fetched.put("io.org", singletonMap("level", "ERROR"));
+        fetched.put("io.org.com", singletonMap("level", "FATAL"));
+
+        // root logger level did not change. The parent logger level did
+        assertTrue(client.parentLogLevelChanged(fetched, ops, Map.entry("io.org.com", "WARN")));
```

Comment:
Should we also add a test that logger from other hierarchy is not affected here? E.g. have a logger `com.org.io` and check that it returns false?